docs/enhancements/template.md

```diff
@@ -26,6 +26,16 @@ To get started with this template:
 The `Metadata` section above is intended to support the creation of tooling
 around the enhancement process.
 
+Make sure that the enhancement covers the following scenarios:
+- REST API and Kubernetes API
+- Late-binding and bind-on-discovery
+- SNO and multinode
+- Day1 (cluster installation) and day2 (adding hosts to an installed cluster)
+- Observability: feature usage, events, status info, etc.
+- Network configuration: static networking vs DHCP, IPv4/v6/dual-stack
+- Supported platforms
```

Comment:
Pls consider including the list of supported platforms to pick from or a link to it. Same for the CPU arch